torch_tvm/operators.h

```diff
@@ -11,17 +17,23 @@ tvm::relay::Expr getOperator(
 bool relayIsNone(tvm::relay::Expr e);
 uint64_t getNoneSentinel();
 
+const std::vector<int32_t>& getParamIndices(torch::jit::Node* node);
+
 using TVMOpFunctor = std::function<tvm::relay::Expr(
     torch::jit::Node* node,
     tvm::Array<tvm::relay::Expr> inputs)>;
 using TVMScheduleFunctor = std::function<const tvm::runtime::PackedFunc*()>;
 
+using ParamIndicesType = std::vector<int32_t>;
```

Comment:
no need for this, it's readable without it

Reply:
Yeah.